# Multi-omics Characterization of the Responses to Seasonal Variations in Diel Cycles in the Marine Picoeukaryote *Ostreococcus tauri*

Dissertation presented by Ana Belén Romero Losada to obtain the PhD Degree by Universidad de Sevilla.

Supervisors:
Prof. Francisco José Romero Campero (@fran-romero-campero in github)
Prof. Mercedes García González

Earth tilted rotation and translation around the Sun produce one of the most pervasive periodic
environmental signals on our planet giving rise to seasonal variations in diel cycles. Marine
phytoplankton undergo significant alterations in response to these signals and, although it
plays a key role on ecosystems, their response to these rhythms remains largely unexplored.
In this work, the marine picoalga Ostreococcus tauri is chosen as model organism grown
under summer long days, winter short days, constant light and constant dark conditions to
characterize these responses. The 80 % of its transcriptome present diel rhythmicity and genes
with robust self-sustained rhythmic expression profiles are identified. A drastic reduction in
proteome rhythmicity with respect to transcriptome rhythmicity is observed with 25 % of the
proteins oscillating. Seasonally specific rhythms are found in key physiological processes
such as cell cycle, photosynthesis, carotenoid biosynthesis, starch accumulation and nitrate
assimilation. A global orchestration between transcriptome, proteome and physiological
dynamics is observed with specific seasonal temporal offsets between gene expression, protein
abundance levels and physiological activity.

The RNA-seq transcriptomic data generated and analysed in this thesis have been deposited
in the Gene Expression Omnibus (NCBI) database and are freely available to the research
community identified with the accession number GSE155535. The SWATH-MS proteomics
data generated in this study have been deposited to the ProteomeXchange Consortium via the
PRIDE (Perez-Riverol et al., 2022) partner repository with the dataset identifier PXD046992.
The shiny app MINOTAUR (Multi-omics INtegration in Ostreococcus TAURi), developed to
facilitate the exploration of the results presented in this thesis can be accessed at:

+ https://greennetwork.us.es/MINOTAUR/
  
The code of MINOTAUR is available from the Github repository:
+ https://github.com/ABRomeroLosada/MINOTAUR

The actual R code developed in this thesis to perform all the analysis is available as a Rmarkdown
report in SANDAL (SeAsonal aNd diel cycles in Ostreoccocus) Github repository:
+ https://github.com/fran-romero-campero/SANDAL

There are other useful tools that have been developed during this thesis.
The codes for MARACAS (MicroAlgae RnA-seq and Chip-seq AnalysiS) pipeline and
AlgaeFUN (microALGAE FUNctional enrichment tool) are freely available from the
following Github repositories:
+ https://github.com/fran-romero-campero/MARACAS
+ https://github.com/fran-romero-campero/AlgaeFUN

